ethsigner/signer/azure/build.gradle

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+jar {
+  baseName 'ethsigner-signer-azure'
+  manifest {
+    attributes(
+      'Specification-Title': baseName,
+      'Specification-Version': project.version,
+      'Implementation-Title': baseName,
+      'Implementation-Version': calculateVersion()
+      )
+  }
+}
+
+
+dependencies {
+
+  implementation project(':ethsigner:commandline')
+  implementation project(':ethsigner:signing-api')
+  implementation project(':ethsigner:signer:file-based')
```

Comment:
do we need this? thought azure was doing signing in the cloud

Reply:
Removed.